Ext4Pkg/Ext4Dxe/Inode.c

```diff
@@ -255,6 +255,44 @@ Ext4FileIsDir (
   return (File->Inode->i_mode & EXT4_INO_TYPE_DIR) == EXT4_INO_TYPE_DIR;
 }
 
+/**
+   Checks if a file is a symlink.
+   @param[in]      File          Pointer to the opened file.
+
+   @return TRUE if file is a symlink.
+**/
+BOOLEAN
+Ext4FileIsSymlink (
+  IN CONST EXT4_FILE  *File
+  )
+{
+  return (File->Inode->i_mode & EXT4_INO_TYPE_SYMLINK) == EXT4_INO_TYPE_SYMLINK;

```

Comment:
For single bits, we generally use != 0 because it's shorter.

Reply:
Oh God, they looked like single Bits to me, but some indeed have two Bits. Yes, agreed.